spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -913,8 +913,14 @@ def highlight_canvas(self, highlight):
         """
         colorname = self.canvas.palette().highlight().color().name()
         if highlight:
-            self.canvas.setStyleSheet(
-                    "FigureCanvas{border: 1px solid %s;}" % colorname)
+            # Highlighted figure is not clear in dark mode with blue color.
+            # See spyder-ide/spyder#10255.
+            if is_dark_interface():
+                self.canvas.setStyleSheet(
+                        "FigureCanvas{border: 2px solid %s;}" % "red")
```

Comment:
Hi @Khalilsqu thanks for working on this

Please use a color of the dark palette while we have  way to retrieve the colors:
https://github.com/ColinDuquesnoy/QDarkStyleSheet/blob/master/qdarkstyle/palette.py#L12

Reply:
Which of these colors is the best to choose as the thumbnail frame outline in dark mode?
The line thickness is set to `2px` instead of 1px 

#505F69
![image](https://user-images.githubusercontent.com/22060413/65244092-aad39300-dae1-11e9-85b6-caf2b86385f7.png)

#32414B
![image](https://user-images.githubusercontent.com/22060413/65244190-de162200-dae1-11e9-9200-ad95e2245c60.png)

#19232D
![image](https://user-images.githubusercontent.com/22060413/65244342-351bf700-dae2-11e9-8f1a-026323492f73.png)

#f0f0f0
![image](https://user-images.githubusercontent.com/22060413/65244437-72808480-dae2-11e9-91bb-1c98ad59e634.png)

#AAAAAA
![image](https://user-images.githubusercontent.com/22060413/65244517-9fcd3280-dae2-11e9-9192-e79f69d58226.png)

#787878
![image](https://user-images.githubusercontent.com/22060413/65244596-d145fe00-dae2-11e9-9888-995026b4bfad.png)

#148CD2
![image](https://user-images.githubusercontent.com/22060413/65244695-0b170480-dae3-11e9-866a-d6c01f5a4408.png)

#1464a0 This is the original color in dark mode except for the thickness (2px now)
![image](https://user-images.githubusercontent.com/22060413/65244866-647f3380-dae3-11e9-9f98-974be4596e45.png)

#14506E
![image](https://user-images.githubusercontent.com/22060413/65245033-b88a1800-dae3-11e9-928f-39d71822e4d5.png)

In my opinion red is more clear than those!

Reply:
I would say use `#148CD2` with 2px thickness

Reply:
And yes `red` is much mor noticeable but it just does not go with the theme.

Reply:
Here the updated one. Way better than before.

![148CD2](https://user-images.githubusercontent.com/22060413/65246003-96919500-dae5-11e9-94bd-ed2f651762b8.gif)